vhdbuilder/packer/components.json

```diff
@@ -41,9 +39,6 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.50",
-        "v1.4.52",
-        "v1.5.23",
         "v1.5.26"
```

Comment:
I think we want to keep at least one v1.4 version since that's used in K8s LTS version 1.27